ui/src/app/kvm/views/KVMDetails/LxdProject/ProjectVMs/VMsTable/StatusColumn/StatusColumn.tsx

```diff
@@ -0,0 +1,53 @@
+import { Spinner } from "@canonical/react-components";
+import { useSelector } from "react-redux";
+
+import DoubleRow from "app/base/components/DoubleRow";
+import machineSelectors from "app/store/machine/selectors";
+import type { Machine } from "app/store/machine/types";
+import { useFormattedOS } from "app/store/machine/utils";
+import type { RootState } from "app/store/root/types";
+import { NodeStatusCode } from "app/store/types/node";
+import { getPowerIcon } from "app/utils";
+
+type Props = {
+  systemId: Machine["system_id"];
+};
+
+const transientStatuses = [
```

Comment:
This appears to be the same as in the machine list, probably worth making this a constant somewhere.

Reply:
Good call. I might make a util like `isTransientStatus` and use that in both places.